src/pattern-library/components/patterns/ColorFoundations.js

```diff
@@ -55,6 +64,14 @@ export default function ColorFoundations() {
         <div className="flex flex-row flex-wrap gap-4">{greyExamples}</div>
       </Library.Pattern>
 
+      <Library.Pattern title="Slates">
+        <p>
+          These slightly blue greys may be used sparingly to help with
+          differentiation and clarity within interfaces.
```

Comment:
Is there specific functional guidance about when to use slate greys (eg. what kinds of elements etc.) or are you still figuring that out?